app/src/main/java/com/gatchimath/totask/TaskListView.java

```diff
@@ -0,0 +1,79 @@
+package com.gatchimath.totask;
+
+import android.content.Context;
+import android.support.v7.widget.DividerItemDecoration;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.AttributeSet;
+import android.view.LayoutInflater;
+import android.view.ViewGroup;
+
+import com.gatchimath.totask.databinding.TaskBinding;
```

Comment:
com.gatchimath.totask.databinding and com.gatchimath.totask are totally separate packages, as Java doesn't have the notion of subpackages.